docs/Reference/plugins/community-plugins/amp-css_es.md

```diff
@@ -0,0 +1,54 @@
+---
+title: AMP CSS
+published: true
+lang: es
+position: 100
+---
+
+# Complementos AMP CSS
+
+<div class="docs-link_table">
+  <a class="homepage" href="https://github.com/pjlamb12/scully-plugin-amp-css"></a>
+  <a class="repository" href="https://github.com/pjlamb12/scully-plugin-amp-css"></a>
+</div>
+
+## Visión general
+
+El propósito del complemento AMP CSS para Scully toma todas las etiquetas `style` de las páginas renderizadas para combinaarlas en una sóla etiqueta `style` en el `head`. Este es un requerimiento que propone Google para las páginas AMP. En [este link](https://www.loom.com/share/35330a858cd741ba92e8be0c0496ffbb) se conversa sobre algunas partes sobre este requerimiento. Scully hace esto relativamiente fácil porque después del renderizado de cada página, un complemento puede ser ejecutado para hacer algo a dicha página; en este caso combinar todos los styles dentro de un sólo tag `style`.
```

Comment:
```suggestion
El propósito del complemento AMP CSS para Scully es tomar todas las etiquetas `style` de las páginas renderizadas para combinarlas en una sola etiqueta `style` en el `head`. Este es un requerimiento que propone Google para las páginas AMP. En [este link](https://www.loom.com/share/35330a858cd741ba92e8be0c0496ffbb) se habla sobre algunas partes sobre este requerimiento. Scully hace esto relativamente fácil porque después del renderizado de cada página, un complemento puede ser ejecutado para hacer algo a dicha página; en este caso combinar todos los styles dentro de un sólo tag `style`.
```

Reply:
Try to don't' translate literally, we need to add sense to paragraph an sentences for the readers